packages/elements/package.json

```diff
@@ -54,6 +54,7 @@
     "@types/prismjs": "1.26.3",
     "@types/resemblejs": "4.1.3",
     "@vitest/browser": "0.34.6",
+    "@vitest/coverage-istanbul": "^0.34.6",
```

Comment:
Any reason to go for istanbul coverage over v8? https://vitest.dev/guide/coverage.html